src/internal/modules/account/controller.js

```diff
@@ -45,7 +45,7 @@ const postCreateAccount = async (request, h) => {
 };
 
 const getSetPermissions = async (request, h, formFromPost) => {
-  const form = formFromPost || setPermissionsForm(request);
+  const form = formFromPost || setPermissionsForm(request, '/account/create-user/set-permissions');
```

Comment:
Here the second argument to`setPermissionsForm` is a URL path 
In the call below it is the request payload. 
Is this correct?